.github/workflows/test.yml

```diff
@@ -88,3 +88,55 @@ jobs:
           git add bricks
           git commit -m "[Chore] Generate Mason bundle"
           git push
+
+  generate_sample_project:
+    # The job generate_sample_project identify that the job generate_mason_bundle must complete successfully
+    # before this generate_sample_project job will run
+    needs: generate_mason_bundle
+    name: Generate the sample project
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - name: Check out
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.WIKI_ACTION_TOKEN }}
+          ref: ${{ github.ref }}
```

Comment:
- `with token` is to checkout repo correctly under bot-nimble account to bypass branch protection https://github.com/nimblehq/flutter-templates/pull/211/files#diff-b6be02f3a9d4c7902a0c7ef15e92a821440068867b5fef0a201f8f254f01e94cR16-R17. I don't think we need it for the bot-account to push a commit to a normal PR.
- why do we need `ref: ${{ github.ref }}`?